examples/intellij-plugin-with-experimental-shared-base/gradle.properties

```diff
@@ -0,0 +1,5 @@
+kotlin.stdlib.default.dependency=false
```

Comment:
Is this change required?

Reply:
Yes, otherwise the plugin blows up trying to load a class from Kotlin stdlib, which is already loaded by bootstrap classloader